www/src/pages/guidelines/logo.js

```diff
@@ -88,7 +88,7 @@ const Guidance = ({ children, image }) => (
     sx={{
       mb: 5,
       mr: 5,
-      width: `calc(50% - 1.25rem)`,
+      width: `calc(50% - ${theme.space[5]})`,
     }}
```

Comment:
We should use [functional values] here and in a couple other places — maybe we can get away without having to import the whole `theme` even? I'll give this a quick shot and follow up here.

Reply:
That's cool, I will keep this in mind. Thank you.